src/engraving/rendering/score/tlayout.cpp

```diff
@@ -5865,9 +5865,18 @@ void TLayout::layoutTempoText(const TempoText* item, TempoText::LayoutData* ldat
         LD_CONDITION(s->ldata()->isSetPos());
     }
 
+    if (!item->autoplace()) {
+        return;
+    }
+
     // tempo text on first chordrest of measure should align over time sig if present, unless time sig is above staff
     Segment* s = item->segment();
-    if (item->autoplace() && s->rtick().isZero()) {
+
+    if (RehearsalMark* rehearsalMark = toRehearsalMark(s->findAnnotation(ElementType::REHEARSAL_MARK, item->track(), item->track()))) {
+        double rightEdge = rehearsalMark->ldata()->bbox().translated(rehearsalMark->pos()).right();
+        const double padding = 0.5 * item->fontMetrics().xHeight();
+        ldata->moveX(rightEdge + padding);
+    } else if (item->autoplace() && s->rtick().isZero()) {
```

Comment:
Checking for `autoplace()` here seems superfluius, doesn't it? We won't get here i